app/assets/v2/scss/grants/hall_of_fame.scss

```diff
@@ -0,0 +1,68 @@
+@import "../lib/bootstrap/functions";
+@import "../lib/bootstrap/variables";
+@import "../lib/bootstrap/mixins";
+
+#hall-of-fame {
+  font-family: 'Libre Franklin', sans-serif;
+  color: $gc-grey-500;
+}
+
+.hof-bg-violet-500 {
+  background-color: $gc-violet-500;
+}
+
+.text-violet-400 {
+  color: $gc-violet-400;
+}
+
+#hof-banner-text {
+  font-size: 20px;
+}
+
+#hall-of-fame h2 {
+  font-size: 24px;
+  margin-bottom: 0px;
+}
+
+@include media-breakpoint-up(sm) {
+  #hall-of-fame h2 {
+    font-size: 32px;
+  }
+}
+
+#hall-of-fame h3 {
+  font-size: 24px;
+  margin-bottom: 0px;
+}
+
+#hall-of-fame h4 {
+  font-size: 20px;
+  margin-bottom: 0px;
+}
+
+#hof-graduated-description {
+  font-size: 16px;
+}
```

Comment:
I recommend avoiding ids for styles in general, for font sizes there are already helpers. The idea on using font size helpers is keep consistency between the design system also being more easy to maintain. 
Also check bootstrap helpers classes.

Reply:
This is fixed.